fjohnson_assignment_4/classy_tiny_instacart_2.rb

```diff
@@ -0,0 +1,188 @@
+module NewUser
+
+    class << self
+       attr_accessor :name, :phone, :address
+    end
+
+    def name
```

Comment:
When you use `attr_accessor` these should be unnecessary, just don't wrap it in `class << self`